source/develop/release-management/features/network/import-vm-net-changes.html.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Change network information in VM import
+category: feature
+authors: yevgenyz
+wiki_category: Feature
+wiki_title: Features/Change network information in VM import
+wiki_revision_count: 1
+wiki_last_updated: 2016-08-29
+feature_name: Change network information in VM import
+feature_modules: engine/api-model
+feature_status: Work in progress for oVirt 4.1
+---
+
+# Change network information in VM import
+
+## Summary
+
+Currently importing a VM from an external source might result in the 
+non-functional VM from the network perspective:
+
+* vNics that are not connected to any network that is defined in the 
+oVirt the VM was imported to or is connected to an undesired vNic 
+profile.
+* The MAC address that was assigned to the vNic in the external 
+system could be problematic in the local oVirt setup.
+The reasons for a MAC to be problematic are:
+
+    * The MAC is already in use by an existing VM.
+    * The MAC is out of range of the destination cluster.
+
+The feature intends to solve the mentioned problems in the flow of 
+importing VMs from a data domain storage.
+
+## Owner
+
+*   Name: [Yevgeny Zaspitsky](User:YevgenyZ)
+*   Email: <yzaspits@redhat.com>
+
+## Proposed solution
+
+### Mapping vNic profiles to the target ones.
+The mapping of the source vNic profiles to the ones that are defined 
+under destination DC/custer would be possible as part of the import 
+operation. 
+
+#### REST API 
+The user would supply the desired mappings as part of the import VMs 
+request. The mapping would look like a collection of entries like the 
+following:
```

Comment:
The mapping would look like a collection of entries like the following:
=>
The mapping would look like a collection of entries: